src/runtime_src/core/common/module_loader.cpp

```diff
@@ -113,65 +102,63 @@ xilinx_xrt()
   return xrt;
 }
 
-// Get list of xclbin repository paths from ini file and append
+// Get list of platform repository paths from ini file and append
 // default repository paths
 std::vector<sfs::path>
-get_xclbin_repo_paths()
+get_platform_repo_paths()
 {
   std::vector<sfs::path> paths;
   
-  // Get repo path from ini file if anya
-  auto repo = xrt_core::config::get_xclbin_repo();
+  // Get repo path from ini file if any
+  auto repo = xrt_core::config::get_platform_repo();
   auto token = std::strtok(repo.data(), ":;");
   while (token) {
     paths.push_back(token);
     token = std::strtok(nullptr, ":;");
   }
 
   // Append default path(s)
-  paths.emplace_back(xrt_core::detail::xclbin_repo_path());
+  const auto default_paths = xrt_core::detail::platform_repo_path();
+  paths.insert(paths.end(), default_paths.begin(), default_paths.end());
```

Comment:
Just for my knowledge, what is this change doing?

Reply:
https://en.cppreference.com/w/cpp/container/vector/insert

This inserts the elements between default_paths.begin() and default_paths.end() at paths.end(). Pretty much appends default paths onto paths in a single line.